server/controllers/listing.js

```diff
@@ -19,13 +19,26 @@ class ListingController {
 
   getAllListings = async (req, res, next) => {
     try {
-      const listing = await this.listingService.getAllListings(req.params);
-   
-      res.status(200).json(listing);
+      const listings = await this.listingService.getAllListings(req.params);
+      res.status(200).json(listings);
     } catch (err) {
       next(ApiError.internal(`${err}`));
     }
   };
+
+  getOneListing = async (req, res, next) => {
```

Comment:
I'm thinking that just `getListing` would be more consistent with other names (such as addListing)

Reply:
I agree :) getListing implies it's one Listing